File: Java/LinearSearch.java
public class LinearSearch {
    public static void main(String[] args) {
        int[] arr = {1, 4, 5, 7, 9};
        int key = 5;
        boolean found = false;
        for (int icout = 0; icout < arr.length; icout++) {
            if (arr[icout] == key) {
                found = true;
                break;
            }
        }
        System.out.println("Element found: " + found);
    }
}

File: cpp/Linear_search.cpp
#include <iostream>
using namespace std;

int main() {
    int size, target;
    cout << "Enter the size of the array: ";
    cin >> size;

    int arr[size];
    cout << "Enter the elements of the array: ";
    for (int i = 0; i < size; i++) {
        cin >> arr[i];
    }

    cout << "Enter the target element to search: ";
    cin >> target;

    int result = -1;
    for (int i = 0; i < size; i++) {
        if (arr[i] == target) {
            result = i;
            break;
        }
    }

    if (result != -1) {
        cout << "Element found at index: " << result << endl;
    } else {
        cout << "Element not found in the array." << endl;
    }

    return 0;
}
